tardis/opacities/macro_atom/macroatom_solver.py

```diff
@@ -66,19 +70,41 @@ def solve_transition_probabilities(
         """
         if self.initialize:
             self.initialize_transition_probabilities(atomic_data)
-
-        transition_probabilities = calculate_transition_probabilities(
-            atomic_data,
-            beta_sobolev,
-            mean_intensities,
+        # I wonder why?
+        # Not sure who wrote this but the answer is that when the plasma is
+        # first initialised (before the first iteration, without temperature
+        # values etc.) there are no j_blues values so this just prevents
+        # an error. Aoife.
+        # Josh followup - I renamed j_blues to mean_intensities to be more agnostic
```

Comment:
Thus we require an estimator for Jb  ji, the mean intensity at frequency ν+  ji – i.e.  in the far blue wing of the transition j → i. This can also be derived from Eq. (43).
Lucy 2003 - I think it might have been defined in Lucy 1999